styles.css

```diff
@@ -0,0 +1,971 @@
+/*Fonts*/
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
```

Comment:
Avoid multiple font usage, single font will serve your purpose, adding multiple fonts will break your app performance, and also it's a bad choice as per UX point of view.

Reply:
This is right but other fonts are only for my brand logo. I think I require these fonts anyway to serve the purpose.